arm-sql/2014-04-01/swagger/connectionPolicies.json

```diff
@@ -0,0 +1,275 @@
+{
+  "swagger": "2.0",
```

Comment:
This is a known bug in examples validator. It thinks that `null` isn't a valid string, instead it expects that `null` will just not be sent on the wire. But this is an existing API that we aren't planning to change, I am documenting it accurately. @veronicagg